config/runner_config.py

```diff
@@ -119,4 +120,11 @@
             target_path=TARGET_PATH,
         ),
     ),
+    CTORGPipeline(
+        path_args=PathArgs(
+            source_path="",
+            masks_path="",
+            target_path=TARGET_PATH,
```

Comment:
Same as in other pipelines
```suggestion
            target_path=TARGET_PATH,
            masks_path="",
```